### ISSUE TITLE
Figure: Tizen .NET architecture on https://docs.tizen.org/application/dotnet/ 

### ISSUE THREAD

**Issue by journey2w** [Samsung/tizen-docs#1187]
As I was looking around Tizen .NET documents, I found that Figure: Tizen .NET architecture on https://docs.tizen.org/application/dotnet/ was changed.

It was changed by you on https://github.com/Samsung/tizen-docs/blob/21dd506a5821d71f5eb4dc524860e999e2a24438/docs/application/dotnet/index.md
BIN +50.9 KB (460%) docs/application/dotnet/media/cs_overview.png 

In my point of view, the dashed line in red in the figure is too stressed.
The previous one was okay, but if you want to insert the red dashed line, it would be great to reduce the width of the line.
Moreover, the image quality is not good because you changed the previous image.
Please update it. 

By the way, I have a PPT file including the architecture figure that I drew.
If you let me change the figure, I can do it.

**Comment by surbhig12** [comment]
@journey2w,

I agree. The developer created the image and I do not have the source files. If you have the PPT then would be great if you could.

**Comment by theojin** [comment]
Closing as stale.

The issue reported a quality problem with the "Tizen .NET architecture" figure on the `.NET` landing page (`docs/application/dotnet/index.md`). That page has since been completely rewritten — the current `index.md` no longer contains the architecture figure or a reference to `cs_overview.png`.

The image itself (`media/cs_overview.png`) is still used on the [Learn About .NET](https://github.com/Samsung/tizen-docs/blob/master/docs/application/dotnet/learn-about-dotnet.md) page. The original complaint about the figure on the old landing page no longer applies because the page has been reorganized. If the image quality on the current page is still a concern, please open a new issue referencing `learn-about-dotnet.md`.

**Comment by theojin** [comment]
Hello, and thank you for filing this issue. We sincerely apologize for the very long delay in responding.

The .NET architecture diagram you mentioned is still in the documentation, but the original source file (PPT) needed to regenerate it at higher quality is no longer available — the original contributor proposed it six years ago and the source material has since been lost.

Unfortunately, without the original source file we are unable to produce an improved version. We are closing this issue as we cannot proceed without the source material. If you are able to provide an updated diagram, we would be happy to incorporate it via a pull request. Thank you for your understanding.
